renderer/js/mtftemplate.js

```diff
@@ -1,42 +1,209 @@
+      <div class='mtf-vert-option <%= MTFController.optionsHeight %>'>\
+        <div class='mtf-vert-ques-option'>\
+          <div class='mtf-vert-ques-text'>\
+            <div class='mtf-vert-ques-text-inner' >\
+              <p onclick='MTFController.showImageModel(event, \"background-image\")' style=\"background-size:100% 100%; background-image:url(<%= val.image %>);\">\
+                \<%= val.text %>\
+                <% if(val.audio){ %> \
+                  <span class='mtf-vert-opt-audio-image' >\
+                    <img src='<%=MTFController.pluginInstance.getAudioIcon('renderer/assets/audio.png') %>' onclick=MTFController.pluginInstance.playAudio({src:'<%= val.audio %>'}) \>\
+                  </span>\
+                <% } %>\
+              </p>\
             </div>\
           </div>\
         </div>\
-      <% });%>\
-    </div>\
-</div>";
+      </div>\
+    <% });%>\
+  </div>\
+  <div class='mtf-vert-container panel panel-body' id='left'>\
+    <% _.each(MTFController.selAns,function(val,key){ %>\
+      <div class='mtf-vert-option <%= MTFController.optionsHeight %>'>\
+        <div class='mtf-vert-ques-option'>\
+          <div class='mtf-vert-ques-text'>\
+            <div class='mtf-vert-ques-text-inner cont-dragula' id='left<%= (key+1) %>' leftindex='<%= val.index %>'><% if(val.selText.length > 0){ %> <p><%= val.selText  %> </p> <% }else{ %><%= val.selText %><% } %></div>\
+          </div>\
+        </div>\
+      </div>\
+    <% });%>\
+  </div>\
+  <div class='mtf-vert-container panel panel-body'>\
+    <% _.each(question.data.option.optionsRHS,function(val,key){ %>\
+      <div class='mtf-vert-option <%= MTFController.optionsHeight %>'>\
+        <div class='mtf-vert-ques-option'>\
+          <div class='mtf-vert-ques-text'>\
+            <div class='mtf-vert-ques-text-inner cont-dragula' id='right<%= (key+1) %>' mapIndex='<%= val.mapIndex %>'><% if(MTFController.selAns[key].selText < 1){ %>\
+              <p onclick='MTFController.showImageModel(event, \"background-image\")' style=\"background-size:100% 100%; background-image:url('<%= val.image %>');\">\
+                <% if(val.audio){ %> \
+                  <span class='mtf-vert-opt-audio-image' >\
+                    <img  src='<%=MTFController.pluginInstance.getAudioIcon('renderer/assets/audio.png') %>' onclick=MTFController.pluginInstance.playAudio({src:'<%= val.audio %>'}) \>\
+                  </span>\
+                <% } %>\
+                <%= val.text %>\
+              </p> \
+              <% } %>\
+            </div>\
+          </div>\
+        </div>\
+      </div>\
+    <% });%>\
+  </div>";
+}
+
+/**
+ * image will be shown in popup
+ * @memberof org.ekstep.questionunit.mtf.mtftemplate
+ */
+MTFController.showImageModel = function (event, imageSrcType) {
+  var eventData;
+  if (imageSrcType == "background-image") {
+    eventData = event.target.style.backgroundImage.slice(4, -1).replace(/"/g, "");
```

Comment:
relaying on style background image path is not good. Redesign/rethink of this workflow.

Reply:
@vinukumar-vs This was already existing, so left this as is. Yes, you are correct, we should not rely on the background image. Will work on a better flow. @sivashanmugam-funtoot